app/src/main/java/org/simple/clinic/selectcountry/adapter/SelectableCountryItem.kt

```diff
@@ -5,24 +5,31 @@ import io.reactivex.subjects.Subject
 import kotlinx.android.synthetic.main.list_selectcountry_country_view.*
 import org.simple.clinic.R
 import org.simple.clinic.appconfig.Country
+import org.simple.clinic.appconfig.displayname.CountryDisplayNameFetcher
 import org.simple.clinic.selectcountry.adapter.Event.CountryClicked
 import org.simple.clinic.widgets.ItemAdapter
 import org.simple.clinic.widgets.recyclerview.ViewHolderX
 
 data class SelectableCountryItem(
     val country: Country,
-    val isSelected: Boolean,
-    val showDivider: Boolean
+    private val isCountryChosenByUser: Boolean,
+    private val showDivider: Boolean,
```

Comment:
Is this how we are doing it? Curious if we are not using `RecyclerView.ItemDecoration` for a reason?

Reply:
I've never used the item decoration feature before so I didn't think about it. I'll revisit this after the feature is complete.